src/views/ProofPriceTagAddMultiple.vue

```diff
@@ -74,19 +83,27 @@ export default {
           value: 2
         }
       ],
+      firstProofUploaded: null,
       proofUploadCount: 0
     }
   },
   computed: {
     getPriceValidationUrl() {
       return '/experiments/price-validation-assistant'
     },
+    getPriceAddMultipleProofIdUrl() {
+      return `/prices/add/multiple?proof_ids=${this.firstProofUploaded.id}`
```

Comment:
need to do another PR to make this work in `PriceAddMultiple`

Reply:
actually the logic was already implemented. but i moved to it the correct view in #1602